src/root.jsx

```diff
@@ -209,15 +217,15 @@ class Root extends React.Component {
         );
     }
 
-    getSpeakerSlotRow(speakerObj, speechTitleObj, card, i) {
+    getSpeakerSlotRow(speakerObj, speakerTitleObj, speechTitleObj, card, i) {
         // The speech number is 1 greater than the speech index.
         const number = i + 1;
         // Make callback specific to this speech number.
         const setSpeech = this.setSpeech.bind(this, number);
 
         return (
             <tr key={i}>
-                <td>Speaker {number}</td>
+                <td>{speakerTitleObj[number - 1]}</td>
```

Comment:
Feature idea: have info bubbles to clarify roles, esp backup speaker and what we expect from someone putting their name down there.

Reply:
let's open that in another issue.